src/components/shared/Button/PaneToggleButton.js

```diff
@@ -6,8 +6,8 @@
 import React, { PureComponent } from "react";
 import classnames from "classnames";
 import Svg from "../Svg";
-import CommandBarButton from "./CommandBarButton";
-import "./PaneToggle.css";
+import { CommandBarButton } from "./";
+import "./styles/PaneToggleButton.css";
```

Comment:
You know, `PaneToggleButton` is cool, but itd be even cooler to do `PaneToggle`

Reply:
Since the only thing this component returns is a button it feels more descriptive to let the person reading know by adding the word Button to the name of the files even though the component already is in a button directory. We might also have a pane toggle reducer and other things that share the same name. By keeping the name button we have all the info we need about the files in the filename accessible at once instead of having to refer to the folder name for additional input as to what PaneToggle might be. 

Your call. I'd stick with PaneToggleButton, but will change it to PaneToggle if you think that fits better.